addons/dns/enable

```diff
@@ -47,13 +47,12 @@ else
   echo "Will use ${nameserver_str} as upstream nameservers"
 fi
 
-DNSIP="10.152.183.10"
-
-DNSIPARG=$2
-if [[ -n "$DNSIPARG" ]]; then
-  DNSIP="$DNSIPARG";
-  echo "Will use ${DNSIPARG} for dns service ip address"
-  echo "Make sure this address is in the service cidr!"
+DNSIP="$2"
+if [ x"${DNSIP}" == "x" ]; then
+  # (QOL): If not using a custom service CIDR, then use the well-known address 10.152.183.10
+  if cat "${SNAP_DATA}/args/kube-apiserver" | grep -q -- '^--service-cluster-ip-range=10.152.183.0/24' 2>/dev/null; then
+    DNSIP="10.152.183.10"
```

Comment:
Are we not trying to remove any kind of hardcodings when assigning IPs?

Reply:
In the default situation where no custom CIDR is set, this saves us from an unnecessary restart of the kubelite service (which would add 1+ minute of initial setup until all services have acquired their lease.